google/cloud/bigtable/README.md

```diff
@@ -42,7 +42,12 @@ Apache 2.0; see [`LICENSE`](../../../LICENSE) for details.
 
 ## Release Notes
 
-### v1.6.x - TBD
+### v1.7.x - TBD
+
+### v1.6.x - 2020-01
+
+* feat: build Chain filter from a range (#3303)
```

Comment:
This two could be merged into a single comment, they are basically related, maybe "* feat: new functions to create Chain and Interleave filters from ranges of Filter objects"

Reply:
Done.